modules/bam2fastq/1.2/bam2fastq.smk

```diff
@@ -93,6 +93,8 @@ if CFG["temp_outputs"]:
         log:
             stdout = CFG["logs"]["fastq"] + "{seq_type}/{sample_id}/bam2fastq.stdout.log",
             stderr = CFG["logs"]["fastq"] + "{seq_type}/{sample_id}/bam2fastq.stderr.log"
+        wildcard_constraints:
+            sample_id = "|".join(mrna_bams)
```

Comment:
Where is mrna_bams defined in the module? This should probably instead be `"|".join(sample_id=CFG["samples"]["sample_id"].tolist())`

Reply:
I had it specified in the header, but I should've put it in the module. I can replace it with what you wrote above ^.